src/EluvioLive.js

```diff
@@ -87,6 +87,23 @@ class EluvioLive {
     };
   }
 
+  /**
+   * Show info about this account.
+   */
+  async AccountShow() {
+    if (!this.client) {
+      throw Error("EluvioLive not intialized");
+    }
+
+    let tenantId = await this.client.userProfileClient.TenantId();
```

Comment:
Would this benefit from running these in parallel?
```js
const [tenantId, walletAddress, userWalletObject, userMetadata] = await Promise.all([
    this.client.userProfileClient.TenantId(),
    this.client.userProfileClient.WalletAddress(),
    this.client.userProfileClient.UserWalletObjectInfo(),
    this.client.userProfileClient.UserMetadata()
]);
```

Reply:
lgtm!  adding